### DISCUSSION TITLE
Is it possible to install Jovian-NixOS and boot off an SD card?

### DISCUSSION THREAD

**Discussion by minego** [Jovian-Experiments/Jovian-NixOS#274] [Q&A] [answered]
I'd really like to try Jovian-NixOS but I'm nervous about installing over top of the normal steam os install.

Is it possible to install and boot off of an SD card so that I can experiment? It also sounds like support for the OLED model is fairly new, and that scares me a bit...

**Comment by K900** [comment, marked as answer]
Yes, it is possible, and the OLED model should work fine.
